### PR TITLE
Former

### PR DESCRIPTION
**Pull request by CJ-Wright**
Add space/key for former members (former and `active: false`, respectively)

### CODE REVIEW

File: regolith/templates/former.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block header %}
+<h1>Former Members</h1>
+{% endblock header %}
+{% block content %}
+{% for p in people if p.get('active', False) %}
```
Comment (scopatz) [inline regolith/templates/former.html]: Shouldn't this be `if not p.get('active', True)`?  That former members are those whose active setting is False, and the default active setting is True?

**Comment by scopatz** [comment]
Thanks @CJ-Wright! I think the people page also needs the active filter so that only current members are selected.

**Comment by scopatz** [comment]
Scratch that last comment, just saw the other PR

**Comment by CJ-Wright** [comment]
@scopatz is that level of docs ok or do you want more?

**Comment by scopatz** [comment]
Yeah that is great!

**Pull request merged**